core/src/test/java/org/elasticsearch/search/suggest/completion/FuzzyOptionsTests.java

```diff
@@ -41,49 +47,54 @@ public static FuzzyOptions randomFuzzyOptions() {
         return builder.build();
     }
 
-    @Override
-    protected FuzzyOptions createTestModel() {
-        return randomFuzzyOptions();
-    }
-
-    @Override
     protected FuzzyOptions createMutation(FuzzyOptions original) throws IOException {
         final FuzzyOptions.Builder builder = FuzzyOptions.builder();
-        builder.setFuzziness(original.getEditDistance())
-            .setFuzzyPrefixLength(original.getFuzzyPrefixLength())
-            .setFuzzyMinLength(original.getFuzzyMinLength())
-            .setMaxDeterminizedStates(original.getMaxDeterminizedStates())
-            .setTranspositions(original.isTranspositions())
-            .setUnicodeAware(original.isUnicodeAware());
+        builder.setFuzziness(original.getEditDistance()).setFuzzyPrefixLength(original.getFuzzyPrefixLength())
+                .setFuzzyMinLength(original.getFuzzyMinLength()).setMaxDeterminizedStates(original.getMaxDeterminizedStates())
+                .setTranspositions(original.isTranspositions()).setUnicodeAware(original.isUnicodeAware());
         switch (randomIntBetween(0, 5)) {
```

Comment:
I've started to appreciate
```
Runnable r = randomFrom(
 () -> builder.setFuzziness....,
 () -> builder.asdfadfasdf
);
r.run();
```

That way you can't forget to change the upper bound when you add a new random option.